dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -12,4 +12,50 @@ def index
   def show
     render json: @section.summarize
   end
+
+  # POST /api/v1/sections
+  # Create a new section
+  def create
+    # Validate login_type
+    # TODO: Push validation into model?
+    login_type = (['none', '', nil].include? params[:login_type]) ? 'email' : params[:login_type]
```

Comment:
It's... sort of validated that way.  An exception is raised when an invalid type is passed - it was uncaught, which was causing unit tests to fail even when I `expect_response :error` and would probably get reported out to Honeybadger (which I'd like to avoid).